src/router/index.ts

```diff
@@ -9,15 +10,8 @@ const router = createRouter({
       name: 'tiny-mce',
       component: TinyMCE
     },
-    // {
-    //   path: '/about',
```

Comment:
The code looks clean and well-structured. The commented-out route has been removed, which is good for maintainability. However, the added comment 'Additional routes can be added here' might be unnecessary as it's self-evident.

``` 
// Additional routes
  ]
```